nifi-extension-bundles/nifi-amqp-bundle/nifi-amqp-processors/src/main/java/org/apache/nifi/amqp/processors/AbstractAMQPProcessor.java

```diff
@@ -303,7 +311,8 @@ protected Connection createConnection(ProcessContext context, ExecutorService ex
         final ConnectionFactory cf = new ConnectionFactory();
         cf.setUsername(context.getProperty(USER).evaluateAttributeExpressions().getValue());
         cf.setPassword(context.getProperty(PASSWORD).getValue());
-
+        cf.setMaxInboundMessageBodySize(context.getProperty(MAX_INBOUND_MESSAGE_BODY_SIZE).evaluateAttributeExpressions().asDataSize(DataUnit.B).intValue());
+        
```

Comment:
Please make sure the build is valid with `mvn clean install -Pcontrib-check`

````
Warning:  src/main/java/org/apache/nifi/amqp/processors/AbstractAMQPProcessor.java:[315] (regexp) RegexpSinglelineJava: Line has trailing whitespace.
````

```suggestion
        cf.setMaxInboundMessageBodySize(context.getProperty(MAX_INBOUND_MESSAGE_BODY_SIZE).evaluateAttributeExpressions().asDataSize(DataUnit.B).intValue());

```